src/rules/web.ts

```diff
@@ -0,0 +1,34 @@
+// Provides dev-time type structures for  `danger` - doesn't affect runtime.
+import {DangerDSLType} from '../../node_modules/danger/distribution/dsl/DangerDSL';
+declare const danger: DangerDSLType;
+
+export declare function message(message: string): void;
+export declare function warn(message: string): void;
+export declare function fail(message: string): void;
+export declare function markdown(message: string): void;
+
+import { Scope } from '../rule.type';
+import { warnIfFilesChanged } from '../utils';
+
+/**
+ * Web rules
+ */
+export let web: Scope = {
+
+  /** Warn if css files were changed */
+  async css() {
+    const files = ([] as string[]).concat(
+      danger.git.created_files  || [],
+      danger.git.deleted_files  || [],
+      danger.git.modified_files || [],
+    );
+
+    for (const file of files) {
+      if (file.match(/\.styl/ig)) {
```

Comment:
I didn't undestand this regex... shound't it be `/.css/` for `css` files?
And I am not sure if this regex is good for React project because we are using styled components which nameing convention is: `.*\.style.ts`

Reply:
@felipewaku That's probably because our web (Angular) projects are using a css preprocessor named [Stylus](http://stylus-lang.com/).

Would it make sense if we had a list of possible CSS variants? e.g.: .css | .styl | .less | .sass...

Reply:
@melloflavio @felipewaku 

> I didn't undestand this regex... shound't it be /.css/ for css files?
> Would it make sense if we had a list of possible CSS variants? e.g.: .css | .styl | .less | .sass...

Yap, you are both right. I'll add more variants before merging, thanks!